src/node_osrm_support.hpp

```diff
@@ -420,13 +420,42 @@ inline bool parseCommonParameters(const v8::Local<v8::Object> &obj, ParamType &p
     if (obj->Has(Nan::New("annotations").ToLocalChecked()))
     {
         auto annotations = obj->Get(Nan::New("annotations").ToLocalChecked());
-        if (annotations->IsBoolean())
+
+        std::string annotations_str = *v8::String::Utf8Value(annotations);
+        if (annotations_str == "true")
+        {
+            params->annotations_type = osrm::RouteParameters::AnnotationsType::All;
+            params->annotations = true;
+        }
+        else if (annotations_str == "false")
+        {
+            params->annotations_type = osrm::RouteParameters::AnnotationsType::None;
+            params->annotations = false;
+        }
+        else if (annotations_str == "duration")
```

Comment:
yeah, it must be similar to what osrm-backend is doing [here for string split by ','](https://github.com/Project-OSRM/osrm-backend/blob/77bab18e6f82f0bc7f01db196c81c7d58a41c2f0/include/server/api/route_parameters_grammar.hpp#L78) and [here mapping strings to bit values](https://github.com/Project-OSRM/osrm-backend/blob/77bab18e6f82f0bc7f01db196c81c7d58a41c2f0/include/server/api/route_parameters_grammar.hpp#L62-L64)